pkg/pillar/evetpm/tpm.go

```diff
@@ -191,7 +196,7 @@ func TpmSign(digest []byte) (*big.Int, *big.Int, error) {
 
 	tpmOwnerPasswd, err := ReadOwnerCrdl()
 	if err != nil {
-		return nil, nil, fmt.Errorf("Error in fetching TPM credentials: %v", err)
+		return nil, nil, fmt.Errorf("fetching TPM credentials failed: %v", err)
 	}
 
 	//XXX This "32" should really come from Hash algo used.
```

Comment:
A number of yetus complaints. Can be seen in the annotated diffs at https://github.com/lf-edge/eve/pull/3315/files

Reply:
Picking on `%v` seems random (not [here](https://github.com/lf-edge/eve/pull/3315/files#diff-9a7252d0ee0de34ffefca7f1d2c6a64c7a1c4b4ccb8da06fb4dba8186bee0953L194) or [here](https://github.com/lf-edge/eve/pull/3315/files#diff-b35ac2eec83e6e132c51f2e1d6d7c2caf8452667cff40df498d57942d7cb4215R280) :/ ). AFAIU `%w` is used when we want to unwrap the error later (calling `errors.Unwrap` on it), but we aren't doing it.

Reply:
I did modified the lines I mentioned above, but strangely no complains from Yetus, I will change all to `%w` even though it same as `%v` in this case. This lint is [disabled](https://github.com/polyfloyd/go-errorlint/blob/ae4007104f25875f59e77af9c55da5700e5d1b5e/errorlint/analysis.go#L32) by default in go-errorlint, maybe we should do the same.